### PR TITLE
local-up-cluster.sh conformance tests - ALLOW_SECURITY_CONTEXT should default to true

### PR DESCRIPTION
**Pull request by dims**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

SecurityContextDeny Admission plugin causes failures in several conformance tests

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by timothysc** [comment]
/assign @liggitt   

I don't use local up.

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ DOCKER_OPTS=${DOCKER_OPTS:-""}
 DOCKER=(docker ${DOCKER_OPTS})
 DOCKERIZE_KUBELET=${DOCKERIZE_KUBELET:-""}
 ALLOW_PRIVILEGED=${ALLOW_PRIVILEGED:-""}
-ALLOW_SECURITY_CONTEXT=${ALLOW_SECURITY_CONTEXT:-""}
+ALLOW_SECURITY_CONTEXT=${ALLOW_SECURITY_CONTEXT:-"true"}
```
Comment (liggitt) [inline hack/local-up-cluster.sh]: the way this is used is odd... it doesn't check "true" or "false", just "empty" vs "not empty". defaulting this to "not empty" means to override, you'd have to do `ALLOW_SECURITY_CONTEXT= hack/local-up-cluster.sh`

Would prefer switching to the same pattern as the other admission plugins: `DENY_SECURITY_CONTEXT_ADMISSION` and if non-empty, add the plugin

Comment (dims) [reply hack/local-up-cluster.sh]: Done! thanks

**Pull request merged**